README.md

```diff
@@ -1,3 +1,11 @@
+<p align="center">
```

Comment:
Given how readmes are rendered on GitHub, center alignment looks weird IMO. Consider aligning this, and the `<h1>`, on the left.